taskweaver/module/event_emitter.py

```diff
@@ -156,7 +157,11 @@ def update_message(self, message: str, is_end: bool = True):
         assert not self.message_is_end, "Cannot update message when update is finished"
         self.post.message += message
         self.message_is_end = is_end
-        self._emit(PostEventType.post_message_update, message, {"is_end": is_end})
+        self._emit(
+            PostEventType.post_message_update,
+            PromptUtil.remove_all_delimiters(message) if is_end else message,
+            {"is_end": is_end},
+        )
```

Comment:
This might be tricky as this would only effectively remove the delimiter if the post is sent in a whole pack, and if sending chunk by chunk, the result would be different even though streaming should semantically transmit the same set of content rather than causing difference.

Reply:
so far, the delimiters can only exist in the message from the code interpreter which is assembled manually. It cannot be generated by the llm, so I cannot image any possibilities of sending chuck by chunk.